app/src/main/java/org/exoplatform/activity/WebViewActivity.java

```diff
@@ -184,6 +185,11 @@ public void onCloseOnBoardingFragment() {
     getSupportFragmentManager().beginTransaction().remove(onBoardingFragment).show(mPlatformFragment).commit();
   }
 
+  @Override
+  public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+    super.onRequestPermissionsResult(requestCode, permissions, grantResults);
+  }
```

Comment:
is this overriding method needed ?

Reply:
Yes. When the user click on Allow or Deny in the permissions request popup, this callback method is called (WebViewActivity.onRequestPermissionsResult), not PlatformWebViewFragment.onRequestPermissionsResult. So we need to override it and call super.onRequestPermissionsResult to call PlatformWebViewFragment.onRequestPermissionsResult.

Reply:
Ok, I thought that the method implementation is the same as super, and it should be inherited from parent class already.